news/scikit-package.rst

```diff
@@ -16,7 +16,7 @@
 
 **Fixed:**
 
-* sub section titles in doc to improve readability and navigation working with conda-forge package
```

Comment:
Removed this news... adding sub headers doesn't seem news-worthy and these are drafts too.